assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -16,35 +16,31 @@ const imagesToItems = images => images.map(
 );
 
 export const GalleryFrontend = ({
-  gallery_block_title,
-  gallery_block_description,
-  className,
-  gallery_block_style,
-  images,
+  attributes,
```

Comment:
Because we `hydrate` the component.

Reply:
Maybe we could set `{}` as default value here? Then we would avoid later on having to check `attributes && attributes.gallery_block_title` for example 🤔